tests/yast2_gui/yast2_keyboard.pm

```diff
@@ -71,8 +73,10 @@ sub run {
     send_key "alt-k";
     wait_still_screen 2;
     send_key "e";
+    send_key "n" if is_sle("=15-SP2");
     send_key_until_needlematch("yast2_keyboard-layout-us", "down");
-    send_key "alt-o";
+    send_key "alt-o" if is_sle("<=15-SP1");
+    send_key "alt-a" if is_sle("=15-SP2");
```

Comment:
I would recommend setting at the beggining:
`my $accept_keybind = is_sle(<=15-SP1) ? "alt-o" : "alt-a";`
and then just use `send_key $accept_keybind;`
since this branching is done more than once for the specific keybind (lines 52-53 as well). 

This can also make the flow a bit more readable, and if for some reason there is need to change the accepting keybind later, there is no danger of forgetting one entry.

Reply:
Thanks for the proposal!